src/main/java/org/takes/facets/fork/FkEncoding.java

```diff
@@ -52,7 +53,7 @@
  *
  * <p>The class is immutable and thread-safe.
  * @author Yegor Bugayenko (yegor@teamed.io)
- * @version $Id$
+ * @version $Id: a38ce59de8810b15af2763d253fd5b2a7833976c $
```

Comment:
@dalifreire remove the unneeded string at the end, this field should be `@version $Id$` only